### ISSUE TITLE
Usage question

### ISSUE THREAD

**Issue by jperasmus** [GoogleChromeLabs/comlink-loader#6]
Excited to try out this webpack loader!

* How does this loader compare to workerize-loader?
* Does this loader require a JS class export or can any other function be exported and used?

**Comment by fnlctrl** [comment]
I've tried both and here's my learnings:

workerize-loader and comlink-loader achieve the same thing - putting code into a worker at compile time. A small difference is that workerize uses individual exported functions in `__webpack_exports__`, while comlink uses a class as a whole.

I chose workerize-loader over comlink-loader because:
- Workerize is smaller and simpler.
- Workerize is ES3, while comlink requires ES6 (Proxy, Map, WeakSet, generator) and ES7 (async functions). This means less build config and smaller bundle size (doesn't require all the polyfill) if you must support some legacy devices (Android 4.4, iOS 9)
